### PR TITLE
fix issuse #37 

### PR DESCRIPTION
**Pull request by zoghal**
fix issuse #37

### CODE REVIEW

File: src/Shell/Task/TestTask.php [hunk 1]
```diff
@@ -201,7 +201,7 @@ public function bake($type, $className)
 
         $baseNamespace = Configure::read('App.namespace');
         if ($this->plugin) {
-            $baseNamespace = str_replace('/', '\\', $this->plugin);
+            $baseNamespace = $this->_pluginNamespace($this->plugin);
```
Comment (bcrowe) [inline src/Shell/Task/TestTask.php]: Trailing whitespace.

**Comment by antograssiot** [comment]
not related but could you also update the composer.json like this so tests passes : 
```
"cakephp/cakephp-codesniffer": "dev-master"
```

I thought the 2.x tag existed at some time...

**Comment by markstory** [comment]
@antograssiot No we never did make that tag. I'll add one now for 2.0.0 :smile:

File: src/Shell/Task/TestTask.php (hunk 1 above)
Comment (markstory) [inline src/Shell/Task/TestTask.php]: Fixed in eb9ff7daa9401e0ee6f95c060b817153ad217994

**Pull request merged**